docs/code-snippets/outlook-snippets.yaml

```diff
@@ -761,6 +779,183 @@ Office.Body.setSelectedDataAsync:
         '<a id="LPNoLP" href="http://www.contoso.com">Click here!</a>',
         {coercionType: Office.CoercionType.Html},
         callback);
+Office.Categories.addAsync_1:
+  - |-
+    var masterCategories = [
+        {
+            "displayName": "Urgent!",
+            "color": Office.MailboxEnums.CategoryColor.Preset0
+        }
+    ];
+
+    Office.context.mailbox.masterCategories.getAsync(getMasterCategoriesCallback);
+
+    Office.context.mailbox.masterCategories.addAsync(masterCategories, function (asyncResult) {
+        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+            console.log("Successfully added categories to master list");
+        } else {
+            console.log("masterCategories.addAsync call failed with error: " + asyncResult.error.message);
+        }
+    });
+
+    Office.context.mailbox.masterCategories.getAsync(getMasterCategoriesCallback);
+
+    var categories = ["urgent!"];
+
+    Office.context.mailbox.item.categories.getAsync(getItemCategoriesCallback);
+
+    Office.context.mailbox.item.categories.addAsync(categories, function (asyncResult) {
+        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+            console.log("Successfully added categories");
+        } else {
+            console.log("categories.addAsync call failed with error: " + asyncResult.error.message);
+        }
+    });
+
+    Office.context.mailbox.item.categories.getAsync(getItemCategoriesCallback);
+
+    Office.context.mailbox.item.categories.removeAsync(categories, function (asyncResult) {
+        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+            console.log("Successfully removed categories");
+        } else {
+            console.log("categories.removeAsync call failed with error: " + asyncResult.error.message);
+        }
+    });
+
+    Office.context.mailbox.item.categories.getAsync(getItemCategoriesCallback);
+
+    Office.context.mailbox.masterCategories.getAsync(getMasterCategoriesCallback);
+
+    Office.context.mailbox.masterCategories.removeAsync(categories, function (asyncResult) {
+        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+            console.log("Successfully removed categories from master list");
+        } else {
+            console.log("masterCategories.removeAsync call failed with error: " + asyncResult.error.message);
+        }
+    });
+
+    Office.context.mailbox.masterCategories.getAsync(getMasterCategoriesCallback);
+
+    function getItemCategoriesCallback (asyncResult) {
```

Comment:
Calling these `get*` is misleading since they just print the data and don't return anything.